components/EventBanner.tsx

```diff
@@ -0,0 +1,37 @@
+// CSS
+import styles from "../styles/components/EventBanner.module.scss";
+// Config
+import * as config from "../config.yaml";
+import ReactMarkdown from "react-markdown";
+
+interface Props {
+  color?: string;
+  contextKey?: string;
+  hidden?: boolean;
+}
+
+export default function EventBanner(
+  { color = "event-banner", contextKey = "bannerInfo", hidden = false }: Props,
+) {
+  const bannerInfo = config[contextKey] || [];
+
+  if (bannerInfo.length === 0 || hidden || bannerInfo[0]?.hidden) {
+    return null;
+  }
+
+  const eventMarkdown = bannerInfo[0].text;
```

Comment:
is there a reason this is an array?

Reply:
`bannerInfo` is an array of results (I don't 100% know why but probably something to do with JavaScript objects) so we need to get the first entry in the array

Reply:
Can you try and remove the `-` in the yaml, I think thats why.

Reply:
Fixed